.travis.yml

```diff
@@ -4,9 +4,8 @@ services:
 
 language: go
 go:
-  - 1.6
-  - 1.7
-  - 1.8
+  - 1.9
+  - 1.10.x
```

Comment:
I guess something went wrong when rebasing on master, as this change has already been applied via https://github.com/SUSE/zypper-docker/commit/93b25244309fbfb951aa75a04256b5a1e0dd64d6.

Reply:
You can revert that with `$ git checkout upstream/master -- .travis.yml` and rebase via `$ git rebase --pull upstream master`.

Reply:
^^^ it's pull --rebase for sure :laughing: